taxonomies/ingredients.txt

```diff
@@ -8,6 +8,8 @@ synonyms:en:hydrolysed, hydrolyzed
 
 synonyms:en:pasteurised, pasteurized
 
+synonyms:en:liquorice, licorice
```

Comment:
there's already an ingredient en:liquorice, licorice, so we do not need to add it as a synonym here.

Reply:
Removed.